evaluate.py

```diff
@@ -101,8 +101,11 @@
         "name": "Non-standard tablespaces",
         "query": "select array_agg(spcname) from pg_tablespace where spcname not in ('pg_default', 'pg_global')",
     }, {
-        "name": "Databases",
-        "query": "select array_agg(datname) from pg_database where datname not in ('template0', 'template1', 'rdsadmin', 'tsadmin')",
+        "name": "Current database",
+        "query": "select current_database()",
+    }, {
+        "name": "Other databases",
+        "query": "select array_agg(datname) from pg_database where datname not in ('template0', 'template1', 'rdsadmin', 'tsadmin', current_database())",
```

Comment:
Instead of two queries, the aim of knowing which database we are viewing can be achieved by adding a "(active)" marker to the respective database.

```sql
defaultdb=> select array_agg(datname) || array[current_database() || '(active)'] from pg_database where datname not in ('template0', 'template1', 'rdsadmin', 'tsadmin', current_database());
                                    ?column?                                     
---------------------------------------------------------------------------------
 {_timescaledb,tsbs_lm,test_is_null,copy_tsbs_lm,no_tsdb,test,defaultdb(active)}
(1 row)
```

Reply:
I see that it works, but I don't understand why we should do it this way.

Reply:
Ah, we are storing this as YAML so "Current database:" makes sense.